samples/apps/github-qna-webapp-react/README.md

```diff
@@ -24,7 +24,12 @@ to get answers about it. The sample highlights how [memory](https://aka.ms/sk/me
 and [embeddings](https://aka.ms/sk/embeddings) work along with the
 [SemanticTextPartitioner](../../../dotnet/src/SemanticKernel/SemanticFunctions/Partitioning/SemanticTextPartitioner.cs)
 when the size of the data is larger than the allowed token limited.
-Each SK function will call Open AI to perform the tasks you ask about.​
+Each SK function will call Open AI to perform the tasks you ask about.
+
+This sample will create embeddings only for markdown files by default in
+order to save time and costs.  You can change the filter by going to
+[GitHubSkill.cs](/semantic-kernel/blob/memory-sample/samples/dotnet/github-skills/GitHubSkill.cs)
```

Comment:
incorrect link, pointing to a file in a branch. The link should be a relative path, starting with `../`